Gruntfile.js

```diff
@@ -34,6 +38,24 @@ module.exports = function (grunt) {
           'src/',
         ],
       },
+      'build-out-wpt': {
+        cmd: 'node',
+        args: [
+          'node_modules/@babel/cli/bin/babel',
+          '--source-maps=false',
```

Comment:
note: source maps are now disabled for the wpt export (which will clean up a lot of useless console warnings). Alternatives: could embed the source inline as a giant base64 blob, or could even include the source files in the export.

Reply:
--source-maps inline would be nice if it doesn't cause any issues

Reply:
It should work fine but the file size is about 4x (webgpu/ 172kB -> 687kB). For that I'm not sure whether it's worth it.

I also just (re?)discovered that using the retainLines option will fix the annoyance where comments were always being printed at the end of the last line, so I guess I'll turn that on.

Reply:
I'm okay either way. Our compiled sources are pretty readable anyway.

Reply:
Okay, I'll leave them disabled for now.